onnxruntime/python/tools/tensorrt/perf/benchmark.py

```diff
@@ -1845,90 +1764,102 @@ def output_latency(results, csv_filename):
     logger.info(f"CUDA/TRT latency comparison are saved to csv file: {csv_filename}")
```

Comment:
In order to prevent this PR from becoming gigantic, I'll have to fix all of these pylint warnings in a separate PR.